src/app/donation-start/donation-start-form/donation-start-form.component.ts

```diff
@@ -1642,7 +1636,6 @@ export class DonationStartFormComponent implements AfterContentChecked, AfterCon
       currencyCode: this.campaign.currencyCode || 'GBP',
       donationAmount: this.donationAmount,
       donationMatched: this.campaign.isMatched,
-      feeCoverAmount: sanitiseCurrency(this.amountsGroup.value.feeCoverAmount),
```

Comment:
this already defaults to zero in matchbot https://github.com/thebiggive/matchbot/blob/73149a162e3db3bf942953b15ea670a59daabbef/src/Application/HttpModels/DonationCreate.php#L36 so we don't need to include it here.